protocol/messenger_profile_showcase.go

```diff
@@ -243,6 +289,7 @@ func (m *Messenger) fromProfileShowcaseCommunityProto(senderPubKey *ecdsa.Public
 }
 
 func (m *Messenger) fromProfileShowcaseAccountProto(messages []*protobuf.ProfileShowcaseAccount) []*identity.ProfileShowcaseAccount {
+	// NOTE: no requests are allowed to be made here
```

Comment:
I saw this commment a lot. I am wondering what it means exactly. I mean what specific lime of code do you mean shouldn't the called for instance and why?

Reply:
extended the comment to : `no requests to the network are allowed to be made here, called in the receiver thread`